src/components/Loading.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import ReactLoading from 'react-loading';
+import colors from '../index.scss';
+
+
+const Loading = (props) => {
+  return (
+    <div className="loading-wrapper">
+      <ReactLoading {...props} />
```

Comment:
If it takes longer than 3 seconds, should we write "It is still loading... Please, be patient."?